src/shared/scrapers/DE/index.js

```diff
@@ -0,0 +1,113 @@
+import * as fetch from '../../lib/fetch/index.js';
+import datetime from '../../lib/datetime/index.js';
+import * as geography from '../../lib/geography/index.js';
+import * as parse from '../../lib/parse.js';
+
+const country = 'iso1:DE';
```

Comment:
You can move this to the scraper object as an attribute

Reply:
I had this on both the scraper object and on the returned objects because sources.md said that country was required on the returned objects, whereas other properties like state were not required if defined on scraper object. Based on your comment it sounds like country is not required, so I've updated sources.md accordingly.